src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -102,6 +101,8 @@ internal bool IsMarshalAsObject
         {
             get
             {
+                object? o = null;
```

Comment:
Some cruft from when you were testing warnings?

Reply:
yes I forgot to revert that.